### PR TITLE
created model for request

### PR DESCRIPTION
**Pull request by gagandeep95**
### What this PR does (required):
- It is a request model which creates a schema for a request model

### CODE REVIEW

File: server/models/Request.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+  user_id: mongoose.Schema.Types.ObjectId,
+  sitter_id: mongoose.Schema.Types.ObjectId,
```
Comment (ezeilo-su) [inline server/models/Request.js:5]: Please `camelCase` these variable names.

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
Looks good! Please fix the comment below.

File: server/models/Request.js (hunk 1 above)
Comment (gagandeep95) [reply server/models/Request.js]: Thanks, @ezeilo-su , I have made these changes

File: server/models/Request.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    required: true,
+  },
+  accepted: {
+    type: Boolean,
+    default: false,
+  },
+  declined: {
+    type: Boolean,
+    default: false,
+  },
```
Comment (ugur-akin) [inline server/models/Request.js:29]: Having two fields to track the status of a booking will often lead to unnecessary complexity and cause more problems down the road. I'd recommend reducing these two to a single field such as `status`. You can maybe throw in an [enum validator](https://mongoosejs.com/docs/schematypes.html#string-validators) there too!

File: server/models/Request.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    ref: "User",
+  },
+  start: {
+    type: Date,
+    required: true,
+  },
+  end: {
+    type: Date,
+    required: true,
+  },
```
Comment (ugur-akin) [inline server/models/Request.js:21]: It would have been nice to see a validation of `end > start` here as a bonus! 😎

**Review by ugur-akin — COMMENTED** [review]
Looks good overall, just a couple of quick suggestions from my end!

File: server/models/Request.js (hunk 2 above)
Comment (gagandeep95) [reply server/models/Request.js:29]: Thanks @ugur-akin , I will make it done.

**Review by ugur-akin — APPROVED** [review]
LGTM!

**Pull request merged**